models/Cargo.toml

```diff
@@ -16,6 +16,6 @@ sqlx = { version = "0.6", features = [
     "offline",
     "postgres",
     "macros",
-    "chrono",
     "json",
 ], optional = true }
+chrono = { version = "0.4.23", features = ["serde"] }
```

Comment:
Postgres already has types for storing time ranges:

https://www.postgresql.org/docs/current/rangetypes.html

https://docs.rs/sqlx-postgres/0.7.0/sqlx_postgres/types/index.html

Reply:
Let's come back and update this change, added a TODO.